Makefile

```diff
@@ -209,8 +209,8 @@ endif
 TEST_FLAGS ?= -v
 .PHONY: unit build-unit-tests
 unit unit-test:
-	curl -L https://go.kubebuilder.io/dl/2.3.1/${GOOS}/${GOARCH} | tar -xz -C /tmp/
-	mv /tmp/kubebuilder_2.3.1_${GOOS}_${GOARCH} /usr/local/kubebuilder
+	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_${os}_${arch}.tar.gz | tar -xz -C /tmp/
+	sudo mv /tmp/kubebuilder_master_${GOOS}_${GOARCH} /usr/local/kubebuilder
```

Comment:
Fix kubebuilder installation in workflow.

Caused by kubernetes-sigs/kubebuilder#2311
Fix was found here: https://github.com/operator-framework/operator-lifecycle-manager/pull/2335/files